forte/utils/utils.py

```diff
@@ -303,3 +304,45 @@ def build_alignment(self, text: str, text_to_align: str):
 
     def _get_opcodes(self):
         yield from self.__matcher.get_opcodes()
+
+
+def create_import_error_msg(
+    extra_module: str,
+    forte_module: str,
+    component_name: str,
+):
+    """
+    Create an error message for importing package extra required by a forte
+    module.
+
+
+    Args:
+        extra_module: module name should be installed by pip.
+        forte_module: forte module User should install by
+            ``pip install forte[`forte_module`]`` to install all
+            extra packages for using the forte module.
+        component_name: the forte component that needs the module.
+
+    """
+    forte_module_link_d = {
+        "data_aug": "https://github.com/asyml/forte/blame/master/README.md#L70",
+        "ir": "https://github.com/asyml/forte/blame/master/README.md#L71",
+        "remote": "https://github.com/asyml/forte/blame/master/README.md#L72",
+        "audio_ext": "https://github.com/asyml/forte/blame/master/README.md#L73",
+        "stave": "https://github.com/asyml/forte/blame/master/README.md#L74",
+        "models": "https://github.com/asyml/forte/blame/master/README.md#L75",
+        "test": "https://github.com/asyml/forte/blame/master/README.md#L76",
+        "wikipedia": "https://github.com/asyml/forte/blame/master/README.md#L77",
+        "nlp": "https://github.com/asyml/forte/blame/master/README.md#L78",
+        "extractor": "https://github.com/asyml/forte/blame/master/README.md#L79",
+    }
+
+    error_msg = (
+        f" `{extra_module}` is not installed correctly."
+        + f" Consider install {extra_module}"
+        + f"via `pip install {extra_module}`."
```

Comment:
Sometimes the module may not be installed via `pip`, just tell the user to install this python module.